ElenaCzubiak-elena-cz/HW_5/course_tracker/config/routes.rb

```diff
@@ -0,0 +1,72 @@
+Rails.application.routes.draw do
+    
+    #get "/enrollments" => "students#enrollments"
+    
+    
+    resources :students do
+        member do 
+            get "/enrollments" => "enrollments#new"
+            post "/enrollments" => "enrollments#create"
+            delete :enrollments
+        end
+    end    
+    
```

Comment:
See if you can clean up these routes so you don't have to use a hash rocket `=>`.